terraform/modules/configs/deploy-all/main.tf

```diff
@@ -441,6 +434,15 @@ module "s3" {
   environment = var.environment
 }
 
+module "iam" {
+  source                       = "../../iam"
+  environment                  = var.environment
+  s3_static_bucket_arn         = module.s3.s3_static_bucket_arn
+  s3_feed_bucket_arn           = module.s3.s3_feed_bucket_arn
+  s3_cnet_bucket_arn           = module.s3.s3_cnet_bucket_arn
+  s3_product_import_bucket_arn = module.s3.s3_product_import_bucket_arn
```

Comment:
Thought there may be a more elegant way to pass these as an array - but wasn't sure

Reply:
Yep I would have thought passing in a `list` type var using the built-in `[obj1,obj1...]` syntax should work - and be compatible with the resource list in the policy statement?

Reply:
The bit I wasn't sure about was appending the `/*` suffix to the arns for this case (assuming a list of arns was passed in)

```
Resource = [
          "${var.s3_static_bucket_arn}/*",
          "${var.s3_feed_bucket_arn}/*",
          "${var.s3_cnet_bucket_arn}/*",
          "${var.s3_product_import_bucket_arn}/*"
        ]
```

Reply:
Ahh yep, missed that!  `formatlist` might help there, whether it's worth it or not.. 🤷‍♂️ 
https://www.terraform.io/docs/language/functions/formatlist.html

Reply:
Missed this in the chaos - good call - just applied that now and it seems to work 👍  Will need to rebase after the other S3 tidy up PR (SINF-394) is merged so will hold off merging until then